docs/INSTALL.md

```diff
@@ -0,0 +1,94 @@
+### Installation (Linux & Mac OS)
+
+#### Recommended installation order
+
+Have [Miniconda](https://conda.io/miniconda.html) or [Anaconda](https://anaconda.org/) installed in the system. The Python 3 version is preferable.
+
+Create conda environment and install required libraries:
+```
+conda create -n wgshgt python=3.5
+source activate wgshgt
+conda install click biopython
+conda install -c biocore scikit-bio
+```
+
+Install third-party applications using conda:
+```
+conda install -c bioconda blast blast-legacy diamond fasttree mafft mcl muscle orthofinder prodigal=2.6.2 raxml t_coffee trimal
+```
+
+Install more third-party applications:
+
+If you are a Ubuntu user, do:
+```
+sudo apt install phyml kalign
```

Comment:
Me neither! I hope that everything can be installed _in situ_ or in `$HOME` without requiring root privilege. I plan to write a script to automate the installation of them (download, compile, export, etc.)